site/src/main/java/uk/nhs/digital/ps/linkprocessors/DefaultDocumentLinkProcessor.java

```diff
@@ -0,0 +1,39 @@
+package uk.nhs.digital.ps.linkprocessors;
```

Comment:
Move out of publication system (ps) package

Reply:
Thanks for the comment. I will move it to the common package